### PR TITLE
fix brightness state for multi profile configs

### PR DESCRIPTION
**Pull request by starkillerOG**
This nasty little bug took me quite some time to figure out.
The brightness was not updating correctly and I was assuming it was caused by the callbacks not comming through, but it turned out the internal state of this upstream lib was just not updated if the adjustment contained more than 1 entry (multi profile config)

Could you please merge and push to a new version as soon as possible?
Then I can finish PR: https://github.com/home-assistant/core/pull/45335

### CODE REVIEW

File: hyperion/client.py [hunk 1]
```diff
@@ -880,7 +880,6 @@ def _update_adjustment(self, adjustment: Optional[List[Dict[str, Any]]]) -> None
             self._serverinfo is None
             or adjustment is None
             or type(adjustment) != list
-            or len(adjustment) != 1
             or type(adjustment[0]) != dict
```
Comment (dermotduffy) [inline hyperion/client.py:883]: Suggestion:
 - Change the `or adjustment is None` check to `or not adjustment` to just reject both None and empty adjustments (len 0).
 - Get rid of the type check for adjustment[0]

**Review by dermotduffy — CHANGES_REQUESTED** [review]
Nice find!

**Comment by dermotduffy** [comment]
BTW: I was planning a new release anyway for some minor changes, so happy to incorporate this PR.

File: hyperion/client.py (hunk 1 above)
Comment (starkillerOG) [reply hyperion/client.py:883]: just a minute

**Comment by starkillerOG** [comment]
@dermotduffy was this what you had in mind?

**Review by dermotduffy — APPROVED** [review]

**Pull request merged**